src/bindings/python/src/openvino/__init__.py

```diff
@@ -27,30 +27,33 @@
 from openvino import properties as properties
 
 # Import most important classes and functions from openvino.runtime
-from openvino.runtime import Model
-from openvino.runtime import Core
-from openvino.runtime import CompiledModel
-from openvino.runtime import InferRequest
-from openvino.runtime import AsyncInferQueue
+from openvino.ie_api import Model
+from openvino.ie_api import Core
+from openvino.ie_api import CompiledModel
+from openvino.ie_api import InferRequest
+from openvino.ie_api import AsyncInferQueue
 
-from openvino.runtime import Symbol
-from openvino.runtime import Dimension
-from openvino.runtime import Strides
-from openvino.runtime import PartialShape
-from openvino.runtime import Shape
-from openvino.runtime import Layout
-from openvino.runtime import Type
-from openvino.runtime import Tensor
-from openvino.runtime import OVAny
+from openvino._pyopenvino import Symbol
+from openvino._pyopenvino import Dimension
+from openvino._pyopenvino import Input
+from openvino._pyopenvino import Output
+from openvino._pyopenvino import Node
+from openvino._pyopenvino import Strides
+from openvino._pyopenvino import PartialShape
+from openvino._pyopenvino import Shape
+from openvino._pyopenvino import Layout
+from openvino._pyopenvino import Type
+from openvino._pyopenvino import Tensor
+from openvino._pyopenvino import OVAny
 
-from openvino.runtime import compile_model
-from openvino.runtime import get_batch
-from openvino.runtime import set_batch
-from openvino.runtime import serialize
-from openvino.runtime import shutdown
-from openvino.runtime import tensor_from_file
-from openvino.runtime import save_model
-from openvino.runtime import layout_helpers
+from openvino.ie_api import compile_model
+from openvino._pyopenvino import get_batch
```

Comment:
is there any logic why some API is in ie_api, while others in _pyopenvino ?

Reply:
Yes,
the ones which are imported from `.ie_api` have `python wrappers` above original binded classes from `._pyopenvino`.

Reply:
ok, but why then it's called ie_api ? better name is requried IMO

Reply:
just legacy internal stuff :)
Ok, we will consider a better name.

Reply:
@mryzhov the name `ie_api` is legacy, the file contains pretty necessary and state-of-the-art code :)

Reply:
Renamed to `_ov_api.py`

Reply:
I think the name of _openvino/runtime/ie.api.py_ shouldn't be renamed in case somebody is using it (even though it's internal)